packages/plugins/src/hardhat/tasks.ts

```diff
@@ -148,6 +158,177 @@ subtask(TASK_CHUGSPLASH_FETCH)
     }
   )
 
+// TODO: Change provider to a non-local RPC provider after demo
```

Comment:
I think best practice here is to use `hre.provider` (or wherever the provider object lives on the config). Then users can run this task with `npx hardhat --network <my network> chugsplash-init`.

Reply:
Fixed this; needed to add @nomiclabs/hardhat-ethers as a dependency

Reply:
I'll approve for now, I might hop in here to figure out how to remove this dependency. I don't wanna have to make downstream projects necessarily use this dep, I assume there are alternatives